src/powershell/install.sh

```diff
@@ -162,6 +163,12 @@ if [ ${#POWERSHELL_MODULES[@]} -gt 0 ]; then
     done
 fi
 
+# If URL for powershell profile is provided, download it to '/opt/microsoft/powershell/7/profile.ps1'
+if [ -n "$POWERSHELL_PROFILE_URL" ]; then
+    echo "Downloading PowerShell Profile from: $POWERSHELL_PROFILE_URL"
+    curl -sSL -o "/opt/microsoft/powershell/7/profile.ps1" "$POWERSHELL_PROFILE_URL"
```

Comment:
Any thoughts if we should configure it for all users vs current user? Looks like the current changes are configuring for all users.

I wonder if we need more community feedbacks for this one!

Reply:
The dev containers are mostly intended for single users and for shared users so I thought it should be fine to add it at system level. 
Another reason I chose above path is that it remains static v/s a new user might be created inside container at a later stage of dev container setup.
If someone really needs to run pwsh without profile they can launch it using `pwsh -noprofile`. But I'm happy to hear community feedback on this.

Reply:
That makes sense, thanks for you explanation.